l10n/en/firefox/all.ftl

```diff
@@ -4,31 +4,58 @@
 
 ### URL: https://www-dev.allizom.org/firefox/all/
 
-firefox-all-check-the-system-requirements = Check the system requirements
-firefox-all-release-notes = Release notes
-firefox-all-source-code = Source code
-firefox-all-need-help = Need help?
-firefox-all-which-browser-would = Which browser would you like to download?
+# HTML page title. Replace "English (US)" with your local language.
+firefox-all-download-the-firefox = Download the { -brand-name-firefox-browser } in English (US) and more than 90 other languages
+
+# HTML page description, also used as the introductory text.
+firefox-all-everyone-deserves-access = Everyone deserves access to the internet — your language should never be a barrier. That’s why — with the help of dedicated volunteers around the world — we make the { -brand-name-firefox-browser } available in more than 90 languages.
```

Comment:
**nit:** avoid older product brand term in new strings

```suggestion
firefox-all-everyone-deserves-access = Everyone deserves access to the internet — your language should never be a barrier. That’s why — with the help of dedicated volunteers around the world — we make the { -brand-name-firefox } browser available in more than 90 languages.
```

Reply:
These show up as altered because I moved them up to the top of the file, they're not new strings.

That said, I might as well fix them while I'm in here.